packages/perseus/src/__tests__/renderer.test.tsx

```diff
@@ -192,12 +188,10 @@ describe("renderer", () => {
                 ...question1,
                 widgets: {
                     ...question1.widgets,
```

Comment:
We're specifically breaking our type formatting here, so we still require the override for this test. 

However, it does make me wonder if there could be some issues with our content with the new, more specific types.